doc/extending/extending_theano_solution_1.py

```diff
@@ -179,8 +179,8 @@ def infer_shape_numpy_dot(node, input_shapes):
 
 @as_op(itypes=[theano.tensor.fmatrix, theano.tensor.fmatrix],
        otypes=[theano.tensor.fmatrix], infer_shape=infer_shape_numpy_dot)
-def numpy_add(a, b):
-    return numpy.add(a, b)
+def np_add(a, b):
```

Comment:
I would keep the old name "numpy_add", it is more clear, and the infer shape function is still named "infer_shape_numpy_add_sub".